zkg

```diff
@@ -2512,6 +2518,12 @@ def argparser():
         " the latest version tag, or if a package has none,"
         ' the default branch, like "main" or "master".',
     )
+    sub_parser.add_argument(
+        "--ignore-dirty-git",
+        action="store_true",
+        help="Allows installation of packages from 'dirty' git clones instead"
+        " of failing.",
```

Comment:
Nit: A way to avoid this weird line break would be to use

```suggestion
        help=(
            "Allows installation of packages from 'dirty' git clones instead of failing."
        ),
```

Here and below.